improver_tests/utilities/test_probability_manipulation.py

```diff
@@ -107,6 +127,34 @@ def expected_inverted_probabilities():
     return np.linspace(1.0, 0.3, 8).reshape(2, 2, 2).astype(np.float32)
 
 
+@pytest.mark.parametrize("above", [True, False])
+@pytest.mark.parametrize(
+    "inequality, etype, inverted_attr",
```

Comment:
I get that `etype` is short, but it took me a moment, and a look down the code, to realise this was probably `expected_type`.  Looking more closely, it only takes two values and behaves like the existing `"above"`, so call this `expected_above` and let it take the values `True, False`. Then your if test on line 152 could be `if expected_above == above:`

Reply:
Done.